docs/modules/mssql.md

```diff
@@ -92,7 +154,8 @@ Deletes both the database and the SQL Server instance.
 
 ##### Provision
 
-Provisions a new SQL Server DBMS only. Databases can be created through subsequent provision requests using the `azure-sql-db-only` service.
+Provisions a new SQL Server DBMS only. Databases can be created through subsequent provision requests using the `azure-sql-database-only` service.
```

Comment:
Suggestion: consider phrasing such as "an empty SQL Server DBMS".  It may be less technically accurate, but I had to think through what "DBMS only" meant in this context.  Or "a SQL Server DBMS instance containing no databases" perhaps?

Reply:
i like it!